cwt/draft-ietf-ace-cbor-web-token.xml

```diff
@@ -126,6 +126,21 @@
         The suggested pronunciation of CWT is the same as the English word
         "cot".
       </t>
+      <section anchor="CBORRelatedTerminology" title="CBOR Related Terminology">
+        <t>
+          In JSON, maps are called objects and only have one kind of map key: a
+          string.  In COSE, we use strings, negative integers and unsigned
+          integers as map keys.  The integers are used for compactness of
+          encoding and easy comparison.  The inclusion of strings allows for an
+          additional range of short encoded values to be used as well.
+        </t>
+        <t>
+          The presence of a key in a COSE map which is not a string or an
+          integer is an error.  Applications can either fail processing or
+          process messages with incorrect keys, however they MUST NOT create
+          messages with incorrect keys.
+        </t>
```

Comment:
Please delete this second paragraph.  There's no reason to discuss invalid CBOR syntax and implementation's responses to it.  The IESG will object to this if we leave it in.